atoma-proxy/src/server/handlers/nodes.rs

```diff
@@ -373,6 +373,8 @@ pub async fn nodes_create_lock(
                     stack_small_id,
                     selected_node_id,
                     tx_digest,
+                    fiat_locked_amount: _, // TODO: fiat
```

Comment:
How come todo fiat ?

Reply:
working on it.

Reply:
I will add the confidential in different PR